src/status_im/multiaccounts/login/core.cljs

```diff
@@ -250,9 +250,7 @@
   [{:keys [db] :as cofx}]
   (let [{:keys [multiaccount multiaccounts :multiaccount/accounts]} db
         {:keys [creating?]}                                         (:multiaccounts/login db)
-        recovering?                                                 (get-in db [:intro-wizard :recovering?])
```

Comment:
is this related to this PR ?

Reply:
Not related to nightly but this is also needed for the referrals program https://github.com/status-im/status-react/issues/11247